clientv3/integration/black_hole_test.go

```diff
@@ -30,6 +30,11 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+// TestServerGRPCKeepAliveTimeout tests ServerParameters keepalive ping on server-side that
+// after having pinged for keepalive check, the server waits for a duration of Timeout and
+// if no activity is seen even after that the connection is closed. The keepalive takes few
```

Comment:
`if no client activity is seen from clients, the connection is closed on server-side`, to make it clear it's server-side operation that's being tested.